src/platform/packages/private/kbn-grid-layout/grid/grid_height_smoother.tsx

```diff
@@ -44,23 +44,8 @@ export const GridHeightSmoother = ({
       smoothHeightRef.current.style.userSelect = 'none';
     });
 
-    /**
-     * This subscription sets global CSS variables that can be used by all components contained within
-     * this wrapper; note that this is **currently** only used for the gutter size, but things like column
-     * count could be added here once we add the ability to change these values
-     */
-    const globalCssVariableSubscription = gridLayoutStateManager.runtimeSettings$
-      .pipe(
-        map(({ gutterSize }) => gutterSize),
-        distinctUntilChanged()
-      )
-      .subscribe((gutterSize) => {
-        smoothHeightRef.current?.style.setProperty('--kbnGridGutterSize', `${gutterSize}`);
-      });
```

Comment:
I moved this subscription to the `useGridLayoutState` callback, because I think it makes more sense there